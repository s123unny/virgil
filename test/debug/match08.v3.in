b ../core/match08.v3 10
run
bt
info l
c
q
