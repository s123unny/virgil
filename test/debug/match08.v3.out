breakpoint|../core/match08.v3 10|0
stop|stopOnBreakpoint
bt|main|../core/match08.v3|10
variable|0|arg|0|int|false
stop|end
result|1
